infrastructure/iac-gip-inclusion/dns/emplois/terraform/locals.tf

```
@@ -1,10 +1,4 @@
-module "dns-emplois" {
-  source = "./dns"
-
-  providers = { scaleway = scaleway.terraform_ci }
-
-  organization_id = var.scw_organization_id
-  managed         = local.managed
+locals {
```

```suggestion
locals {
    emplois_root_zone = "emplois.inclusion.gouv.fr"
```

J'ai meme un doute sur l'override des locals, pas sur que ce ne soit pas semantiquement mieux de faire un :

```
locals = {
    emplois = {
        root_zone = "emplois.inclusion.gouv.fr"
        records = { ... }
    }
}
```

---

Et si tu veux declarer les records avec le sous-domaine `emplois.` inclus tu peux passer ta root_zone `inclusion.gouv.fr` en variable a chaque module

---

Du coup là pas certain de comprendre, la "root zone" c'est une zone qui n'en est pas vraiment une parmi l'API zones. C'est le domaine racine en fait.

On est partis du principe que l'on n'aurait qu'une seule zone, et que comme tout serait géré depuis l'IaC et compartimenté ici, que l'on ne définirait pas plusieurs zones DNS. Mais p-e ais-je mal compris ce point lors de mes échanges avec François, chaud d'en rediscuter si pas clair.